### ISSUE TITLE
[Adaptive Cards> Community Call - January 10, 2019]: Ensures <iframe> and <frame> elements have an accessible name.

### ISSUE THREAD

**Issue by vaishali1397** [microsoft/AdaptiveCards#8219]
### Target Platforms

Other

### SDK Version

1.5

### Application Name

Adaptive Cards

### Problem Description

**Test Environment:**
OS Version: 22H2 (Build 25267.1000)
Browser: Edge Dev (110.0.1556.0 (Official build) dev (64-bit)
URL: https://adaptivecards.io/blog/2019/Community-Call-January/
Tool: Accessibility Insights for Web

**Repro Steps:**

1. Open the above URL in edge dev browser.
2. Run the fast pass of Accessibility Insights for Web
3. Observe the issue.

**Actual Result:**
Element has no title attribute or aria-label attribute does not exist or is empty or aria-labelledby attribute does not exist, references elements that do not exist or references elements that are empty

**Expected Result:**
Ensures <iframe> and <frame> elements have an accessible name.

**User Impact:**
Screen reader user will have difficulty in interacting with the contents and in understanding the purpose if accessible name is not defined.

**MAS Reference link:**
https://microsoft.sharepoint.com/:w:/s/accessibility/EZBde1C58eFKkdEvHy1sfB0BbfrE-sgqsVO5g2zVY6wbDA?e=jEWXd4

Have feedback to share on Bugs? Please help fill [Trusted Tester Bug Feedback (office.com)](https://nam06.safelinks.protection.outlook.com/?url=https://forms.office.com/pages/responsepage.aspx?id%3Dv4j5cvGGr0GRqy180BHbR5dUqGQgJ9lItv-jqY8F09lURVdYTENIRkY2UzlCR0NKV1E2VUE5OVpDVy4u&data=04%7c01%7cv-chisha%40microsoft.com%7c1334b110f98744a7f5df08d968eb842b%7c72f988bf86f141af91ab2d7cd011db47%7c1%7c0%7c637656183104980596%7cUnknown%7cTWFpbGZsb3d8eyJWIjoiMC4wLjAwMDAiLCJQIjoiV2luMzIiLCJBTiI6Ik1haWwiLCJXVCI6Mn0%3D%7c1000&sdata=r4ndSHk/LrBDwzi7FO%2BjKN0DCnqUSUcELBvGRkgc5CU%3D&reserved=0)"

### Screenshots

<img width="953" alt="MAS4 1 2 Ensure iframe or frame has an accessible name" src="https://user-images.githubusercontent.com/104483005/211288597-d28d1e88-88e8-467d-93e1-2bd8ed3105a6.png">
<img width="773" alt="MAS4 1 2 Frame does not have accessible name" src="https://user-images.githubusercontent.com/104483005/211289285-3bb41b82-d6fa-4ad7-9a48-93e4c3796f9d.png">

### Card JSON

```JSON
NA
```

**Comment by vagpt** [comment]
Hi @anna-dingler,

As this issue is still repro, so could you please let us know is there any future plan to fix this issue?


**Snippet:**

![image](https://github.com/microsoft/AdaptiveCards/assets/95913748/9c9c0765-d970-45ab-8be4-cd41ca7cc00c)

**Comment by anna-dingler** [comment]
Hi @vagpt,

This issue is fixed on our internal site. I expect us to make another website release this month, so I will go ahead and mark this as completed.

Please let me know if you have any questions!
